SU-HSDO/suhumsci
# Hsd8 641 capx limits

# READY FOR REVIEW

# Summary
- Take a given cap url and break it into multiple pages to allow for larger data sets. Each url will be 25 profiles.

# Needed By (Date)
- asap

# Urgency
- medium

# Steps to Test
1. `git checkout 8.1.0-release`
1. `composer install --prefer-source`
1. `blt drupal:sync --site=symsys --partial`
1. `drush @symsys.local mim custm_symsys_capx --update`
1. validate only 100 profiles are updated
1. `git checkout HSD8-641-capx-limits`
1. `drush @symsys.local cr`
1. `drush @symsys.local mim custm_symsys_capx --update`
1. validate more than 100 have been updated.
1. `drush @symsys.local cget migrate_plus.migration.custm_symsys_capx source.urls --include-overridden`
1. verify multiple urls with increasing page numbers: `p=1`, `p=2`, etc

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
